src/components/card/MyReservationCard.tsx

```diff
@@ -0,0 +1,107 @@
+import Image from "next/image";
+import { HTMLAttributes } from "react";
+
+import { cn } from "@/utils/classNames";
+import { formatNumber } from "@/utils/common";
+
+interface MyReservationCardProps extends HTMLAttributes<HTMLDivElement> {
+  activity: {
+    bannerImageUrl: string;
+    title: string;
+  };
+  status: "pending" | "confirmed" | "declined" | "canceled" | "completed";
+  reviewSubmitted: boolean;
+  totalPrice: number;
+  headCount: number;
+  date: string;
+  startTime: string;
+  endTime: string;
+}
+
+/**
+ * 예약관리 리스트 아이템
+ */
+export default function MyReservationCard({
+  status,
+  activity,
+  date,
+  startTime,
+  endTime,
+  totalPrice,
+  headCount,
+  reviewSubmitted,
+  className = "",
+}: MyReservationCardProps) {
+  /* TODO: Button component 반영 */
+  function ReviewButtons() {
+    return reviewSubmitted ? (
+      <>
+        <button className="w-[100%]">예약 변경</button>
+        <button className="w-[100%]">예약 취소</button>
+      </>
+    ) : (
+      <button className="w-[100%]">후기 작성</button>
+    );
+  }
+  return (
+    <li className="grid gap-[20px]">
```

Comment:
하나의 의견입니다! 😅

```suggestion
    <article className="grid gap-[20px]">
```

카드의 경우 하나의 독립적인 컨텐츠 단위를 의미하는 `article`로 반영하면 어떨까 싶습니다! 🤔
`li` 태그는 단순한 목록을 의미한다고 생각해서, 카드라는 복합적인 컨텐츠를 담는다는 의미보다 좀 더 그 의미가 맞을 거라고 생각해요!

그래서 실제로 사용하는 쪽에서 아래처럼 사용하는 것이 제일 명확하지 않을까 합니다 하핳 😆

```tsx
<ul>
  <li>
    <MyReservationCard />
  </li>
  <li>
    <MyReservationCard />
  </li>
</ul>
```